setup.json

```diff
@@ -95,7 +95,7 @@
         ],
         "pre-commit": [
             "astroid<2.5",
-            "mypy==0.790",
+            "mypy==0.910",
```

Comment:
Is this dependency bump necessary for implementing the automation of `__all__`?

Reply:
Well, one of the primary purposes of this PR was to get the new version of mypy working, so I wanted to make sure that was actually the case